Bio/PDB/internal_coords.py

```diff
@@ -18,6 +18,159 @@
 description of a structure for 3D printing, and reading/writing structures as
 internal coordinate data files.
 
+**Usage:**
+::
```

Comment:
This next block of code is just an RST literal.

Were there complications using Python syntax highlighting?

Reply:
No problems, the code is pretty much identical to what's in the Tutorial (except better annotated there).  I'm just cynical and expect no one will look past the API docs so wanted the sample code there at the top of the main internal coords file.